GTFS.DB.SQLite/Collections/SQLiteStopCollection.cs

```diff
@@ -347,6 +347,27 @@ public IEnumerable<Stop> Get()
             });
         }
 
+        /// <summary>
+        /// Returns entity ids
+        /// </summary>
+        /// <returns></returns>
+        public IEnumerable<string> GetIds()
```

Comment:
Where are most of these being used? This should always be faster obviously than fetching the whole table and then just taking the ids. The one small caveat would be the ID generators. Since they only force evaluation when you request an ID, not on creation it can actually be slower to use this method when creating the ID generator (say in the Route Wizard). We could try refactor so these are only evaluated lazily but I think that was a bit more tricky. Just bear in mind when using it.

Reply:
Cool, that makes sense. Probably no need to change the ID generators to use this for now. It's nice that they are created in <1ms even if it means the first call takes a bit longer.